CONTRIBUTING.md

```diff
@@ -2,10 +2,33 @@
 
 First of all, thanks for the decision to contribute to DHIS2! :heart:
 
+To contribute to any of the DHIS2 repositories, please follow a few guidelines.
+
+- Follow Code Standards
+- Use semantic commit messages
+- Follow contribution guidelines
+- Common sense!
```

Comment:
I kind of get what this is referring to, but also a bit odd that it's not mentioned again later in the text. Perhaps we could be a bit more specific. Common sense in what way?